src/adapter/src/coord/sequencer/cluster.rs

```diff
@@ -574,13 +585,20 @@ impl Coordinator {
                 logging,
                 idle_arrangement_merge_effort,
                 replication_factor,
+                disk,
             }) => {
                 use AlterOptionParameter::*;
                 match &options.size {
                     Set(s) => *size = s.clone(),
                     Reset => coord_bail!("SIZE has no default value"),
                     Unchanged => {}
                 }
+                match &options.disk {
+                    Set(d) => *disk = *d,
+                    // TODO(guswynn|pH14): this should be `upsert_source_disk_default`
```

Comment:
I don't think so? Shouldn't this just always be `false`? Unless we introduced some new `cluster_disk_default` variable, but I don't think there's any need to add a TODO for that. It's not obviously necessary!

Reply:
Oh, just read your TODOs in the PR description.

Reply:
yeah, this pr is supposed to be a draft! i messed up!